src/modules/dashboard/components/SmiteDialog/SmiteDialogForm.tsx

```diff
@@ -309,15 +319,17 @@ const SmiteDialogForm = ({
             elementOnly
             maxButtonParams={{
               fieldName: 'amount',
-              maxAmount: String(userPercentageReputation || 0),
+              maxAmount: String(unformattedUserReputationAmount),
               setFieldValue,
             }}
             disabled={inputDisabled}
           />
-          <div className={styles.percentageSign}>%</div>
-          <p className={styles.inputText}>{`max: ${
+          <div className={styles.percentageSign}>pts</div>
+          <p
+            className={styles.inputText}
+          >{`max: ${formattedUserReputationAmount} pts (${
             userPercentageReputation === null ? 0 : userPercentageReputation
-          }%`}</p>
+          }%)`}</p>
```

Comment:
Please refactor this to use `FormattedMessage` and values. It will be much more readable.